src/components/ConversationsSelect.ts

```diff
@@ -6,6 +6,13 @@ import { FC } from '..';
 export interface ConversationSelectProps extends ContainerProps<string> {
   initialConversation?: string;
   actionId: string;
+  filter?: {
+    include?: ('im' | 'mpim' | 'private' | 'public')[];
```

Comment:
what's the difference between these? obviously 'public' is public, but the other three?

Reply:
public -> public channels
private -> private channels
im -> direct message with just two people
mpim -> multi-person direct message, a group chat

the block fields can be seen here https://api.slack.com/reference/block-kit/composition-objects#filter_conversations